src/python/director/continuouswalkingdemo.py

```diff
@@ -153,7 +153,7 @@ def _setupOnce(self):
         lcmUtils.addSubscriber('FOOTSTEP_PLAN_RESPONSE', lcmdrc.footstep_plan_t, self.onFootstepPlanContinuous)# additional git decode stuff removed
         lcmUtils.addSubscriber('IHMC_FOOTSTEP_STATUS', lcmdrc.footstep_status_t, self.onFootstepStatus)
         lcmUtils.addSubscriber('EST_ROBOT_STATE', lcmdrc.robot_state_t, self.onRobotStatus)
-        stepParamsSub = lcmUtils.addSubscriber('ATLAS_STEP_PARAMS', lcmdrc.atlas_behavior_step_params_t, self.onAtlasStepParams)
+        stepParamsSub = lcmUtils.addSubscriber('ATLAS_STEP_PARAMS', atlas.behavior_step_params_t, self.onAtlasStepParams)
```

Comment:
I haven't had a chance to try running it, but I'm pretty sure you need to "import atlas" in all the files, not just atlasdriver.py.  It won't throw errors until the code is executed, and not all of this code is executed on start up so you wouldn't notice it right away.